src/controllers/queryRunner.ts

```diff
@@ -736,6 +717,148 @@ export default class QueryRunner {
             );
         }
 
+        await this.writeStringToClipboard(copyString);
+    }
+
+    public async sendToClipboard(
+        data: DbCellValue[][],
+        batchId: number,
+        resultId: number,
+        selection: ISlickRange[],
+        headersFlag,
+    ) {
+        let copyString = "";
```

Comment:
Is this entire function something new you created in this PR? IIRC some of the code here duplicates with the existing copyResults function, could we re-use (possibily by extracting out some common pieces) instead of duplicating them?